frontend/src/app/data-entry/episode-form/episode-entities-form/episode-entities-form.component.html

```diff
@@ -0,0 +1,48 @@
+<ng-container *ngIf="data$ | async as data">
+    <ng-container *ngIf="data.episode">
+        <ng-container *ngIf="(linkedEntities$ | async)?.length; else noEntities">
+            <p>The following {{entityName}}s are involved in this episode:</p>
+            <ul class="list-unstyled vstack gap-4">
+                <li *ngFor="let entity of linkedEntities$ | async">
+                    <lc-episode-link-form [entityType]="entityType" linkTo="entity"
+                            [entityID]="entity.id"
+                            [episodeID]="data.episode.id">
+                        <button removeControl class="btn btn-danger"
+                            [attr.aria-label]="'remove ' + entityName + ': ' + entity.name"
+                            [ngbTooltip]="'remove this ' + entityName"
+                            (click)="removeEntity$.next(entity.id)">
+                            <lc-icon [icon]="actionIcons.remove" />
+                        </button>
+                    </lc-episode-link-form>
+                </li>
+            </ul>
+        </ng-container>
+        <ng-template #noEntities>
+            <p>
+                No {{entityName}}s have been added to this episode.
+            </p>
+        </ng-template>
+    </ng-container>
+</ng-container>
+
+<div class="mt-4 mb-4">
+    <div ngbDropdown>
+        <button type="button" class="btn btn-primary"
+            #dropdownToggle
+            [attr.id]="dropdownToggleID"
```

Comment:
Is this diffferent from `[id]` you use elsewhere?

Reply:
No. `id` is one of the few HTML attributes that Angular supports without `attr.`, but because most HTML attributes require `attr.*`, I sometimes type that out of habit.